app/src/main/res/layout/component_history.xml

```diff
@@ -25,8 +26,17 @@
         layout="@layout/recently_closed_nav_item"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
```

Comment:
Let's move all of these margin changes into a separate issue. I don't want to mix this PR with a UX change and prefer to deal with it in a separate PR and have a proper UX review.

Reply:
I think it's better to keep it. We are adding a new button to the screen, the inconsistency is obvious, the change is pretty small and there will be an UX review anyway

Reply:
The reason I am asking this to move separately is because I don't think the implementation is correct. I think you just want to add a View that has a height of 8dp instead of selectively adding the margin into each of these components. If both `recently_closed_nav_item` and `synced_history_nav_item` are both visible, then you now have a 8dp spacing in between them, which you can see in the Figma should not be the case.

<img width="418" alt="Screen Shot 2022-04-06 at 6 44 02 PM" src="https://user-images.githubusercontent.com/1190888/162084668-3b1f560e-4334-4a77-9f82-76fcd0037c91.png">

I also don't see this 8dp spacing in the other "Library" UI (Bookmark or Download) so I think we can also ignore it based on Nicole's comment https://github.com/mozilla-mobile/fenix/pull/24374/files#r847585856.